src/paginator.js

```diff
@@ -0,0 +1,300 @@
+import assign from 'lodash.assign';
+import omit from 'lodash.omit';
+
+
+export const styles = Object.freeze({
+  pageNumber: 1,
```

Comment:
> Because the value of pageNumber is not at all a number of a page. It is an int representing a style of pagination.

Oo I see. It's a style that just happens to be called "pageNumber". Got it, thanks!